util/packaging/docker/README.md

```diff
@@ -0,0 +1,145 @@
+# Chapel Docker Image
+
+## What is Chapel?
+
+[Chapel](https://chapel-lang.org/) is a parallel programming language designed
+for productivity at scale. It is developed as
+[open source on GitHub](https://github.com/chapel-lang/chapel/).
+
+
+## Quickstart
+Chapel hello world in Docker:
+```
+$ echo 'writeln("Hello, world!");' > hello.chpl
+
+$ docker run --rm -it -v "$PWD":/myapp -w /myapp chapel/chapel /bin/bash
+root@xxxxxxxxx:/myapp# chpl -o hello hello.chpl
+root@xxxxxxxxx:/myapp# ./hello
+Hello, world!
+```
+
+Compile and run a Chapel program using the Docker container as a
+one-shot, so you can develop outside of the container:
+```
+$ echo 'writeln("Hello, world!");' > hello.chpl
+$ docker run --rm -v "$PWD":/myapp -w /myapp chapel/chapel chpl -o hello hello.chpl
+$ docker run --rm -v "$PWD":/myapp -w /myapp chapel/chapel ./hello
+Hello, world!
+```
+
+
+## Docker Image Tags
+
+### [`chapel/chapel:<version>`](https://hub.docker.com/r/chapel/chapel)
+Supported Chapel versions:
+* [`1.28.0`, `latest`](https://github.com/chapel-lang/chapel/releases/tag/1.28.0)
+* [`1.27.0`](https://github.com/chapel-lang/chapel/releases/tag/1.27.0)
+* [`1.26.0`](https://github.com/chapel-lang/chapel/releases/tag/1.26.0)
+* [`1.25.1`](https://github.com/chapel-lang/chapel/releases/tag/1.25.1)
+* [`1.25.0`](https://github.com/chapel-lang/chapel/releases/tag/1.25.0)
+
+
+This is the core image for Chapel. It provides the complete Chapel compiler and
+runtime.  It can be used to compile and run Chapel programs inside the Docker
+container. On 64-bit Linux hosts, the compiled Chapel program binary can
+sometimes be executed outside the container (your mileage may vary).
+Other Chapel-based Docker images can be created from this image.
+
+### [`chapel/chapel-gasnet:<version>`](https://hub.docker.com/r/chapel/chapel-gasnet/)
+* `1.27.0`, `latest`
+* `1.26.0`
+* `1.25.1`
+* `1.25.0`
+
+The Chapel core image (above), rebuilt with `CHPL_COMM=gasnet` and
+`GASNET_SPAWNFN=L`. Simulates a multilocale Chapel platform within the Docker
+container.
```

Comment:
Maybe a sentence about what this means?  Something like, "Multiple locales in Chapel programs are used to take advantage of distributed memory parallelism between nodes in a cluster.  Since programs compiled with Chapel using the above would live within a single Docker container, they would not actually experience parallel computation, but the simulation of it could help with local debugging."